extension/api.js

```diff
@@ -32,18 +47,30 @@ const DEFAULT_SEARCH_SETTING_NAME = "defaultSearch";
 // If the value exists, we have ran before and will not run again.  If
 // we showed the panel, the value will be true.  We do not track what
 // the result of showing the panel is (ie. the user accepted or not).
```

Comment:
We do not track the user decision in the code (we track it via telemetry), but should we start doing so? If we need another iteration of this, we could bail out early if we know the previous user decision.

Reply:
Would we actually show the panel again to a user who previously said no?  or yes?

Reply:
At this point, anything is possible. Storing the decision at least gives us the option not to prompt them again. We might not use it after all, but it's better to have it and not need it than he other way around.

Reply:
Ok, the telemetry reason is now stored in the pref